sqlmesh/core/audit/definition.py

```diff
@@ -152,7 +152,8 @@ def audit_map_validator(cls: t.Type, v: t.Any) -> t.Dict[str, t.Any]:
     if isinstance(v, (exp.Tuple, exp.Array)):
         return dict(map(_maybe_parse_arg_pair, v.expressions))
     elif isinstance(v, dict):
-        return v
```

Comment:
we need to pass in the pydantic values dictionary, look at how model/meta.py is done, and let's double check that it's passed through. the audit definition should have the model dialect implicitly passed in and then when do values.get("dialect")

Reply:
ok thanks for the hint I'll look at how that's done